webapi/src/main/scala/org/knora/webapi/responders/admin/GroupsResponderADM.scala

```diff
@@ -304,79 +299,58 @@ final case class GroupsResponderADMLive(
-                        )
+      projectADM <- findProjectByIriOrFail(
+                      request.project.value,
+                      NotFoundException(
+                        s"Cannot create group inside project <${request.project}>. The project was not found."
                       )
-
-        // check the custom IRI; if not given, create an unused IRI
-        customGroupIri: Option[SmartIri] = createRequest.id.map(_.value).map(iri => iri.toSmartIri)
-        groupIri <- iriService.checkOrCreateEntityIri(
-                      customGroupIri,
-                      GroupIri.makeNew(Shortcode.unsafeFrom(projectADM.shortcode)).value
                     )
 
-        /* create the group */
-        createNewGroupSparqlString =
-          sparql.admin.txt
-            .createNewGroup(
-              AdminConstants.adminDataNamedGraph.value,
-              groupIri,
-              groupClassIri = OntologyConstants.KnoraAdmin.UserGroup,
-              name = createRequest.name.value,
-              descriptions = createRequest.descriptions.value,
-              projectIri = createRequest.project.value,
-              status = createRequest.status.value,
-              hasSelfJoinEnabled = createRequest.selfjoin.value
-            )
-
-        _ <- triplestore.query(Update(createNewGroupSparqlString))
+      // check the custom IRI; if not given, create an unused IRI
+      customGroupIri: Option[SmartIri] = request.id.map(_.value).map(iri => iri.toSmartIri)
+      groupIri <- iriService.checkOrCreateEntityIri(
+                    customGroupIri,
+                    GroupIri.makeNew(Shortcode.unsafeFrom(projectADM.shortcode)).value
+                  )
+
+      /* create the group */
+      createNewGroupSparqlString =
+        sparql.admin.txt
+          .createNewGroup(
+            AdminConstants.adminDataNamedGraph.value,
+            groupIri,
+            groupClassIri = OntologyConstants.KnoraAdmin.UserGroup,
+            name = request.name.value,
+            descriptions = request.descriptions.value,
+            projectIri = request.project.value,
+            status = request.status.value,
+            hasSelfJoinEnabled = request.selfjoin.value
+          )
 
-        /* Verify that the group was created and updated  */
-        createdGroup <-
-          groupGetADM(groupIri)
-            .flatMap(ZIO.fromOption(_))
-            .orElseFail(UpdateNotPerformedException(s"Group was not created. Please report this as a possible bug."))
+      _ <- triplestore.query(Update(createNewGroupSparqlString))
 
-      } yield GroupGetResponseADM(createdGroup)
+      /* Verify that the group was created and updated  */
+      createdGroup <-
+        groupGetADM(groupIri)
+          .flatMap(ZIO.fromOption(_))
+          .orElseFail(UpdateNotPerformedException(s"Group was not created. Please report this as a possible bug."))
```

Comment:
```suggestion
          .someOrFail(UpdateNotPerformedException(s"Group was not created. Please report this as a possible bug."))
```

Reply:
Will improve it in follow up PR related to `/admin/groups`.